sshuttle/client.py

```diff
@@ -678,13 +672,21 @@ def matches_acl(dstip, dstport, store_to_check):
 
     return False
 
+def get_trusted_fqdn():
+    try:
+        if os.environ['TRUSTED_DOMAIN_FQDN']:
```

Comment:
I don't think you need this condition, it should work the same way without it

Reply:
Correct. Updated